<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PenyuluhForm</class>
 <widget class="QWidget" name="PenyuluhForm">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>500</width><height>380</height></rect>
  </property>
  <property name="windowTitle">
   <string>Form Penyuluh (Sederhana)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     
<item row="0" column="0"><widget class="QLabel" name="lblId"><property name="text"><string>ID Penyuluh</string></property></widget></item>
<item row="0" column="1"><widget class="QSpinBox" name="id_penyuluh"><property name="enabled"><bool>false</bool></property><property name="maximum"><number>2147483647</number></property></widget></item>

<item row="1" column="0"><widget class="QLabel" name="lblNama"><property name="text"><string>Nama</string></property></widget></item>
<item row="1" column="1"><widget class="QLineEdit" name="nama"/></item>

<item row="2" column="0"><widget class="QLabel" name="lblAlamat"><property name="text"><string>Alamat</string></property></widget></item>
<item row="2" column="1"><widget class="QLineEdit" name="alamat"/></item>

<item row="3" column="0"><widget class="QLabel" name="lblStatus"><property name="text"><string>Status</string></property></widget></item>
<item row="3" column="1"><widget class="QComboBox" name="status">
 <item><property name="text"><string>PNS</string></property></item>
 <item><property name="text"><string>Honorer</string></property></item>
</widget></item>

<item row="4" column="0"><widget class="QLabel" name="lblSubsektor"><property name="text"><string>Subsektor</string></property></widget></item>
<item row="4" column="1"><widget class="QComboBox" name="subsektor">
 <item><property name="text"><string>Tanaman Pangan</string></property></item>
 <item><property name="text"><string>Hortikultura</string></property></item>
 <item><property name="text"><string>Perkebunan</string></property></item>
 <item><property name="text"><string>Peternakan</string></property></item>
 <item><property name="text"><string>Perikanan</string></property></item>
</widget></item>

<item row="5" column="0"><widget class="QLabel" name="lblMulai"><property name="text"><string>Tanggal Mulai</string></property></widget></item>
<item row="5" column="1"><widget class="QDateEdit" name="tgl_mulai"><property name="calendarPopup"><bool>true</bool></property></widget></item>

    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item><widget class="QPushButton" name="btnSave"><property name="text"><string>Simpan</string></property></widget></item>
     <item><widget class="QPushButton" name="btnClear"><property name="text"><string>Bersihkan</string></property></widget></item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation"><enum>Qt::Horizontal</enum></property>
       <property name="sizeHint"><size><width>40</width><height>20</height></size></property>
      </spacer>
     </item>
     <item><widget class="QPushButton" name="btnClose"><property name="text"><string>Tutup</string></property></widget></item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>